content/rocur/guide/index.en.md

```diff
@@ -54,6 +54,10 @@ Please follow the [Twitter terms of service](https://help.twitter.com/en/rules-a
   
   
 ## Tweeting Tips  
+
+### Notes from a previous curator
+
+[Curating for @WeAreRLadies on Twitter: From creating content to cultivating connections.](https://blog.rstudio.com/2021/09/23/curating-for-wearerladies-on-twitter/) by Shannon Pileggi, Sep 23, 2021.
```

Comment:
Thank you! :rocket:  Could we use the link to the post on your personal website instead?